### ISSUE TITLE
Not a known account class hash

### ISSUE THREAD

**Issue by CharGitWork** [xJonathanLEI/starkli#33]
Hello, I am a beginner and I have this error when I try to create a descriptor account from a local node `starknet-devnet-rs` would it be possible to understand the nature of this error?
ty
![Capture d'écran 2023-09-12 100112](https://github.com/xJonathanLEI/starkli/assets/133235269/00aee69b-3569-45c1-b706-33f3657a2eea)

Edit: Here's a concrete example of what I get with an address provided by the local node launch:

![Capture d'écran 2023-09-12 113654](https://github.com/xJonathanLEI/starkli/assets/133235269/012ae205-56e9-480f-80d1-2a0a52e0a882)

**Comment by xJonathanLEI** [comment]
Starkli relies on your account class hash to understand what the account contract type is, which relies on a list of known account contract class hashes to function.

This error means that the default account created for you by `starknet-devnet-rs` simply hasn't been registered with Starkli yet. You need to ask `starknet-devnet-rs` maintainers to register the account class here. Otherwise, you can also hack it through by manually modifying an OpenZeppelin account file.

**Comment by CharGitWork** [comment]
Okay thanks, to be honest I think it would be nice if in the future there were examples of simple documentation... for example being able to simply use a temporary local nodes like `katana` proposes, given the place Starkli seems to be taking in the ecosystem.
Thanks for your reply.

**Comment by veoquynhs** [comment]
> Starkli relies on your account class hash to understand what the account contract type is, which relies on a list of known account contract class hashes to function.
> 
> This error means that the default account created for you by `starknet-devnet-rs` simply hasn't been registered with Starkli yet. You need to ask `starknet-devnet-rs` maintainers to register the account class here. Otherwise, you can also hack it through by manually modifying an OpenZeppelin account file.

I'm facing the same error too. Can you explain in a more explicit way how to solve this problem? I think this problem might due to the fact that Cairo 1 has just arrived to Argent X. After upgrading my account to Cairo 1, I cannot deploy my contract: 
![image](https://github.com/xJonathanLEI/starkli/assets/132073919/9a20664d-1c52-4636-85e4-73ba33ff3944)

Then I delete my starkli-wallets folder, which contain the account and keystore.json, when I run starkli account fetch, I got the same error

**Comment by CharGitWork** [comment]
> > Starkli relies on your account class hash to understand what the account contract type is, which relies on a list of known account contract class hashes to function.
> > This error means that the default account created for you by `starknet-devnet-rs` simply hasn't been registered with Starkli yet. You need to ask `starknet-devnet-rs` maintainers to register the account class here. Otherwise, you can also hack it through by manually modifying an OpenZeppelin account file.
> 
> I'm facing the same error too. Can you explain in a more explicit way how to solve this problem? I think this problem might due to the fact that Cairo 1 has just arrived to Argent X. After upgrading my account to Cairo 1, I cannot deploy my contract: ![image](https://user-images.githubusercontent.com/132073919/267540246-9a20664d-1c52-4636-85e4-73ba33ff3944.png)
> 
> Then I delete my starkli-wallets folder, which contain the account and keystore.json, when I run starkli account fetch, I got the same error ![image](https://user-images.githubusercontent.com/132073919/267540457-8a97e65f-04d3-4c61-aca6-626a54cf127b.png)

I'm not in a position to help you, but you should know that I've had this same error several times, if I remember correctly.

By the way I still can't get `starkli` to work with `katana` or `starknet-devnet-rs`, there are probably things I don't understand even after reading the documentation.
 
Can anyone help me, I'm looking to set up a simple and functional local development node.
It would be much appreciated

**Comment by xJonathanLEI** [comment]
@veoquynhs I wasn't aware that Argent upgraded to Cairo 1. Will work on supporting that tomorrow. Thanks for flagging.

**Comment by xJonathanLEI** [comment]
@veoquynhs Argent Cairo 1 account contract is now supported (in [v0.1.14](https://github.com/xJonathanLEI/starkli/releases/tag/v0.1.14)). Upgrade your Starkli installation with `starkliup` and try the fetch command account.

@CharGitWork It's definitely a goal for Starkli to be able to work with accounts from local nodes. I will take a look at what account contracts they're using and add them to Starkli. It's a bit tough to catch up though as they could update their account contracts anytime.

**Comment by xJonathanLEI** [comment]
I looked into it and it seems `starknet-devnet-rs` and `katana` use the same OZ v0.5.0 account contract. I've added (https://github.com/xJonathanLEI/starkli/commit/5a5cc1aff40f79f540d99263bb255700d1d36a36) this class to the known account list and it can now be fetched. Note that this commit hasn't been released so unless you install from source this won't be available until the next release is out.
